tests/Report/JunitReporterTest.php

```diff
@@ -0,0 +1,285 @@
+    }
+
+    public function testGenerateWithTimeAndMemory()
+    {
+        $expectedXml = <<<'XML'
+<?xml version="1.0" encoding="UTF-8"?>
+<testsuites>
+  <testsuite name="PHP CS Fixer" tests="1" assertions="1" failures="1" errors="0" time="1.234">
+    <testcase name="someFile" file="someFile.php" assertions="1">
+      <failure type="code_style">Wrong code style</failure>
+    </testcase>
+  </testsuite>
+</testsuites>
+XML;
+
+        $actualXml = $this->reporter->generate(
+            ReportSummary::create()
+                ->setChanged(
+                    array(
+                        'someFile.php' => array(
+                            'appliedFixers' => array('some_fixer_name_here'),
+                        ),
+                    )
+                )
+                ->setMemory(2.5 * 1024 * 1024)
+                ->setTime(1234)
+        );
+
+        $this->assertJunitXmlSchema($actualXml);
+        $this->assertXmlStringEqualsXmlString($expectedXml, $actualXml);
+    }
+
+    public function testGenerateComplex()
+    {
+        $expectedXml = <<<'XML'
+<?xml version="1.0"?>
+<testsuites>
+  <testsuite assertions="3" errors="0" failures="3" name="PHP CS Fixer" tests="2" time="1.234">
+    <testcase assertions="2" file="someFile.php" name="someFile">
+      <failure type="code_style">applied fixers:
+---------------
+* some_fixer_name_here_1
+* some_fixer_name_here_2
+
+Diff:
+---------------
+
+this text is a diff ;)</failure>
+    </testcase>
+    <testcase assertions="1" file="anotherFile.php" name="anotherFile">
+      <failure type="code_style">applied fixers:
+---------------
+* another_fixer_name_here
+
+Diff:
+---------------
+
+another diff here ;)</failure>
+    </testcase>
+  </testsuite>
+</testsuites>
+XML;
+
+        $actualXml = $this->reporter->generate(
+            ReportSummary::create()
+                ->setAddAppliedFixers(true)
+                ->setChanged(
+                    array(
+                        'someFile.php' => array(
+                            'appliedFixers' => array('some_fixer_name_here_1', 'some_fixer_name_here_2'),
+                            'diff' => 'this text is a diff ;)',
+                        ),
+                        'anotherFile.php' => array(
+                            'appliedFixers' => array('another_fixer_name_here'),
+                            'diff' => 'another diff here ;)',
+                        ),
+                    )
+                )
+                ->setMemory(2.5 * 1024 * 1024)
+                ->setTime(1234)
+        );
+
+        $this->assertJunitXmlSchema($actualXml);
+        $this->assertXmlStringEqualsXmlString($expectedXml, $actualXml);
+    }
+
+    /**
+     * Validates generated xml report with schema.
+     * Uses JUnit XML schema from Jenkins.
+     *
+     * @link https://svn.jenkins-ci.org/trunk/hudson/dtkit/dtkit-format/dtkit-junit-model/src/main/resources/com/thalesgroup/dtkit/junit/model/xsd/junit-7.xsd
```

Comment:
@link is deprecated in PSR5 https://github.com/phpDocumentor/fig-standards/blob/master/proposed/phpdoc.md#710-link-deprecated would you mind changing it to @see?